pennylane/ops/qutrit/parametric_ops.py

```diff
@@ -0,0 +1,176 @@
+# Copyright 2018-2022 Xanadu Quantum Technologies Inc.
+
+# Licensed under the Apache License, Version 2.0 (the "License");
+# you may not use this file except in compliance with the License.
+# You may obtain a copy of the License at
+
+#     http://www.apache.org/licenses/LICENSE-2.0
+
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+# pylint: disable=too-many-arguments
+"""
+This submodule contains the discrete-variable quantum operations that are the
+core parameterized gates for qutrits.
+"""
+import itertools
+
+import numpy as np
+
+import pennylane as qml
+from pennylane.operation import Operation
+from pennylane.ops.qutrit.observables import THermitian
+
+
+class TRX(Operation):
+    r"""
+    The single qutrit X rotation
+
+    Performs the RX operation on the specified 2D subspace. The subspace is
+    given as a keyword argument and determines which two of three single-qutrit
+    basis states the operation applies to.
+
+    The construction of this operator is based on section 3 of
+    `Di et al. (2012) <https://arxiv.org/abs/1105.5485>`_.
+
+    .. math:: TR_x^{jk}(\phi) = \exp(-i\phi\sigma_x^{jk}/2),
+                \sigma_x^{jk} = |j\rangle\langle k| + |k\rangle\langle j|,
+                j, k \in \{0, 1, 2\}, j \neq k
+
+    **Details:**
+
+    * Number of wires: 1
+    * Number of parameters: 1
+    * Number of dimensions per parameter: (0,)
+
+    Args:
+        phi (float): rotation angle :math:`\phi`
+        wires (Sequence[int] or int): the wire the operation acts on
+        subspace (Sequence[int]): the 2D subspace on which to apply operation
+        do_queue (bool): Indicates whether the operator should be
+            immediately pushed into the Operator queue (optional)
+        id (str or None): String representing the operation (optional)
+
+    **Example**
+
+    The specified subspace will determine which basis states the operation actually
+    applies to:
+
+    >>> qml.TRX(0.5, wires=0, subspace=[0, 1]).matrix()
```

Comment:
For the sake of being concise, perhaps I can add a reference to the qubit `RX` class in the documentation to avoid redundancy? It doesn't seem to be common practice to include the action of an operator on the basis states. Let me know what you think and I can update the documentation accordingly.